nltk/draw/util.py

```diff
@@ -90,7 +90,7 @@ class CanvasWidget(metaclass=ABCMeta):
     The current value of an attribute value can be queried using the
     ``__getitem__`` operator:
 
-        >>> cn['color']
+        >>> print(cn['color'])
```

Comment:
Is this necessary? Does it not correctly convert to string otherwise or something?

Reply:
When I remove the print() the test fails: 

`Failed example:
    cn['color']
Expected:
    red
Got:
    'red'
`

Reply:
Oh, that makes sense! It's expecting a variable, not a string, because that's what we've put as the expected answer. Changing `red` to `"red"` should also work, but it does not matter much.